### ISSUE TITLE
Invalid JSON message when attempting to view Notebook

### ISSUE THREAD

**Issue by ScoobyQ** [prakharrathi25/artificial-intelligence-for-trading#13]
The following notebook [tree_shap_solution.ipynb ](https://github.com/prakharrathi25/artificial-intelligence-for-trading/blob/master/part2_ai_algorithms_in_trading/quizzes_and_exercises/21_feature_importance/tree_shap_solution.ipynb) is no longer viewable and there is an invalid JSON message showing instead.

**Comment by prakharrathi25** [comment]
Yeah, they might have updated the course material or changed an access token. It's an old repository and I don't have access to the course anymore.
